src/displayapp/Apps.h

```diff
@@ -42,14 +43,31 @@ namespace Pinetime {
       Weather
     };
 
+    enum class WatchFace : uint8_t {
+      Digital = 0,
+      Analog = 1,
+      PineTimeStyle = 2,
+      Terminal = 3,
+      Infineat = 4,
+      CasioStyleG7710 = 5,
```

Comment:
It's unnecessary to assign these values.

```suggestion
      Digital,
      Analog,
      PineTimeStyle,
      Terminal,
      Infineat,
      CasioStyleG7710,
```

Reply:
On the code perspective, you are right, it's not necessary to assign specific values to the field. However, since those specific value are part of the specification of the protocol, I think it's safer to make them explicit. This way, we'll think twice before changing them.

Reply:
You are right, I removed those assignations.